### DISCUSSION TITLE
signifikansi vs adj.r.squared dalam linear regression

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#720] [Q&A] [answered]
Saya membuat 2 model. Model 1 terdapat prediktor yang tidak signifikan dan Model 2 seluruh prediktor signifikan.
Akan tetapi Adj.R.Squared model 1 jauh lebih baik. 
Pertanyaannya adalah model mana yang lebih baik digunakan?

Adj.R.Squared model 1 = 0.73
Adj.R.Squared model 2 = 0.64

Model 1
![image](https://github.com/teamalgoritma/community/assets/23212456/6f4b0eb0-6d7d-47ca-a4c6-0c5d5fce4c41)

Model 2
![image](https://github.com/teamalgoritma/community/assets/23212456/7ebdef2c-a3f8-42c3-a465-ed10bfd37194)

**Comment by arkanayudhistira** [comment, marked as answer]
Dear @Gustana,

Berdasarkan perbandingan hasil yang diperoleh oleh kedua model tersebut, model yang lebih baik adalah model pertama, yang memiliki nilai R-Squared lebih tinggi. Alasannya adalah signifikansi prediktor hanya menunjukkan seberapa baik korelasi prediktor tersebut terhadap target. Namun, yang model yang baik adalah model yang secara keseluruhan merepresentasikan data dengan baik, tidak berdasarkan kualitas masing-masing prediktor saja. Sehingga, nilai R-Squared merupakan acuan pembanding performa model yang lebih baik.

Semoga membantu.
